app/clinic/index.html

```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="../../bower_components/keyboard/dist/css/keyboard-previewkeyset.min.css" />
     <link rel="stylesheet" href="../../bower_components/angular-datepicker/dist/angular-datepicker.min.css">
     <link rel="stylesheet" href="../../bower_components/bootstrap-switch/dist/css/bootstrap3/bootstrap-switch.min.css"/>
-    <link rel="stylesheet" href="../../bower_components/angular-chart.js/dist/angular-chart.css" />
+
     <link rel="stylesheet" href="../../bower_components/nvd3/build/nv.d3.min.css"/>
```

Comment:
For now we will be using Angular Charts, 
I left NVD3 there because I'm still working with it, because there are some features that ChartJS lacks, whithin 2 weeks once I complete other testing tasks I will pull the NVD3 version.